fractions/flyway/src/main/java/org/wildfly/swarm/flyway/FlywayFraction.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.flyway;
+
+import org.wildfly.swarm.spi.api.Fraction;
+import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
+import org.wildfly.swarm.spi.api.annotations.DeploymentModules;
+
+/**
+ * Flyway Fraction
+ * 
+ * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
+ */
+@DeploymentModules({
+        @DeploymentModule( name = "org.flywaydb"),
```

Comment:
I'm not completely sure, and not worth redoing this PR, but I think since you have export=true in the module.conf, you don't need to add org.flywaydb module to the export.

But I could be wrong.  Worth experimenting in a subsequent round.  But as said, not worth refiguring this PR.